indico/web/forms/validators.py

```diff
@@ -379,6 +379,14 @@ class SoftLength(Length):
     where surrounding whitespace is stripped before validation.
     """
 
+    def __call__(self, form, field):
+        orig_data = field.data
+        field.data = re.sub(r'(\r\n|\r)', '\n', field.data)
```

Comment:
I think `field.data` might be `None` in some cases - at least the original Length validator takes that into account, so maybe safer to do so here as well..

```suggestion
        if field.data is not None:
            field.data = re.sub(r'(\r\n|\r)', '\n', field.data)
```